packages/components/src/JSONSchemaRenderer/JSONSchemaRenderer.component.js

```diff
@@ -98,32 +144,45 @@ function typeResolver(schema) {
 			throw new UnkownTypeException(type);
 		}
 
-		return renderer(e[0], title, e[1], schema);
+		if (uiSchema && uiSchema[e[0]] && uiSchema[e[0]]['ui:widget'] === 'hidden') {
```

Comment:
Can you move that just after the `if (!schema[e[0]]) {` so that the code to get the type renderer is not executed if the field is hidden ?